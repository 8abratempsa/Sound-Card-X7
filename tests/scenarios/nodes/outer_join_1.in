octosql "SELECT * FROM range(start=>-10, end=>3) l
                    OUTER JOIN range(start=>-2, end=>6) r ON l.i = r.i
                    OUTER JOIN range(start=>-20, end=>20) rr ON l.i + r.i = rr.i"
