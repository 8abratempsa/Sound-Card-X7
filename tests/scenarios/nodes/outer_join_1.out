+--------+--------+--------+
|  l.i   |  r.i   |  rr.i  |
+--------+--------+--------+
| <null> | <null> |    -20 |
| <null> | <null> |    -19 |
| <null> | <null> |    -18 |
| <null> | <null> |    -17 |
| <null> | <null> |    -16 |
| <null> | <null> |    -15 |
| <null> | <null> |    -14 |
| <null> | <null> |    -13 |
| <null> | <null> |    -12 |
| <null> | <null> |    -11 |
| <null> | <null> |    -10 |
| <null> | <null> |     -9 |
| <null> | <null> |     -8 |
| <null> | <null> |     -7 |
| <null> | <null> |     -6 |
| <null> | <null> |     -5 |
| <null> | <null> |     -3 |
| <null> | <null> |     -1 |
| <null> | <null> |      1 |
| <null> | <null> |      3 |
| <null> | <null> |      5 |
| <null> | <null> |      6 |
| <null> | <null> |      7 |
| <null> | <null> |      8 |
| <null> | <null> |      9 |
| <null> | <null> |     10 |
| <null> | <null> |     11 |
| <null> | <null> |     12 |
| <null> | <null> |     13 |
| <null> | <null> |     14 |
| <null> | <null> |     15 |
| <null> | <null> |     16 |
| <null> | <null> |     17 |
| <null> | <null> |     18 |
| <null> | <null> |     19 |
| <null> |      3 | <null> |
| <null> |      4 | <null> |
| <null> |      5 | <null> |
|    -10 | <null> | <null> |
|     -9 | <null> | <null> |
|     -8 | <null> | <null> |
|     -7 | <null> | <null> |
|     -6 | <null> | <null> |
|     -5 | <null> | <null> |
|     -4 | <null> | <null> |
|     -3 | <null> | <null> |
|     -2 |     -2 |     -4 |
|     -1 |     -1 |     -2 |
|      0 |      0 |      0 |
|      1 |      1 |      2 |
|      2 |      2 |      4 |
+--------+--------+--------+
